src/main/java/blusunrize/immersiveengineering/common/fluids/IEFluid.java

```diff
@@ -106,11 +106,14 @@ public boolean isSame(@Nonnull Fluid fluidIn)
 		return fluidIn==entry.getStill()||fluidIn==entry.getFlowing();
 	}
 
-	//TODO all copied from water. Maybe make configurable?
 	@Override
 	public int getTickDelay(LevelReader p_205569_1_)
 	{
-		return 5;
+		// viscosity delta to water (1000)
+		int dW = this.getFlowing().getAttributes().getViscosity()-1000;
```

Comment:
Why not use grab the viscosity for water straight from the fluid attributes for `Fluids.WATER`?

Reply:
Hadn't thought about it tbh. >_>